src/ApiTemplates/BoredAPICaller.js

```diff
@@ -3,6 +3,7 @@ import React, { useEffect } from "react";
 
 function BoredAPICaller(props) {
   const expressServerBoredApiUrl = "http://localhost:5000/boredApi";
+  const { isSubmitButtonPressed } = props; // Destructuring
```

Comment:
you could take this a step further and destructure it right in the function signature if you want instead by doing 
```
function BoredAPICaller({ isSubmitButtonPressed }) { ...
```

Reply:
So I did that, however would you recommend the other props within this component? Because it kinda looks ugly because there is prop argument and then isSubmitted argument